mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -643,9 +657,12 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 		ARG 1 cameraX
 		ARG 3 cameraY
 		ARG 5 cameraZ
+	METHOD m_rykuwyoq getPosition (F)Lnet/minecraft/unmapped/C_hynzadkk;
+		ARG 1 pos
```

Comment:
Is that argument really a position?

Isn't it some kind of tickDelta thing?
This needs double-checking.